cmd/gossamer/main.go

```diff
@@ -61,15 +67,65 @@ func main() {
 	}
 }
 
+func LvlFromInt(lvlUint int) (log.Lvl, error) {
```

Comment:
This can be replaced by `log.Lvl(lvl int)`. This just casts the `int` to a `Lvl` which is an alias for `int`